src/pipeline.ts

```diff
@@ -208,11 +241,18 @@ export async function checkNewCandidateMatch(
 }
 
 async function findSearchableTorrents() {
-	const { torrents } = getRuntimeConfig();
+	const { torrents, dataDirs } = getRuntimeConfig();
 	let parsedTorrents: Searchee[];
 	if (Array.isArray(torrents)) {
 		const searcheeResults = await Promise.all(
-			torrents.map(createSearcheeFromTorrentFile)
+			torrents.map(createSearcheeFromTorrentFile) //also create searchee from path
+		);
+		parsedTorrents = searcheeResults
+			.filter((t) => t.isOk())
+			.map((t) => t.unwrapOrThrow());
+	} else if (dataDirs && dataDirs.length > 0) {
```

Comment:
This brings up a couple questions:

Naturally we want to avoid excess searchees, so we should try to filter out duplicates from .torrents and data. How is this done and which do we prioritize? If I understand the db correctly, the searchee's name field determines if something has been searched for or not. Should we filter based on that?

Also, does this mean we want to apply the risky/safe logic to regular torrents? The code for risky matching relies on linking which might not be something we want for .torrent searches. We could only apply risky based on if the searchee has a path or not.

Reply:
> we should try to filter out duplicates from .torrents and data. How is this done and which do we prioritize?

this is what I've been thinking about as well. I think at some point it may be beneficial to merge them, but for now we could just prioritize one. It probably doesn't matter much but for the sake of reducing change, I'd probably lean towards prioritizing torrents.

Reply:
Can you clarify the prioritization? If both data and a .torrents dir are specified, pick the .torrents (info log as well saying dataDirs is ignored), and only do one on a given search?